lib/graphql/railtie.rb

```diff
@@ -1,12 +1,15 @@
 # frozen_string_literal: true
+
 module GraphQL
   class Railtie < Rails::Railtie
-    config.before_configuration do
-      # Bootsnap compile cache has similar expiration properties,
-      # so we assume that if the user has bootsnap setup it's ok
-      # to piggy back on it.
-      if ::Object.const_defined?("Bootsnap::CompileCache::ISeq") && Bootsnap::CompileCache::ISeq.cache_dir
-        Language::Parser.cache ||= Language::Cache.new(Pathname.new(Bootsnap::CompileCache::ISeq.cache_dir).join('graphql'))
+    config.graphql = config.active_storage = ActiveSupport::OrderedOptions.new
```

Comment:
```suggestion
    config.graphql =  ActiveSupport::OrderedOptions.new
```
? I don't get why we assign `active_storage` here.

Reply:
🤦 Sorry, not sure how that got there